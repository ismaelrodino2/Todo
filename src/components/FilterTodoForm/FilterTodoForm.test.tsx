import { render } from '@testing-library/react';
import { FilterTodoForm } from '.';

describe('Tests for FilterTodoForm component', () => {
  it('Should match snapshot', async () => {
    const { container } = render(<FilterTodoForm />);

    expect(container).toMatchSnapshot();
  });
});
