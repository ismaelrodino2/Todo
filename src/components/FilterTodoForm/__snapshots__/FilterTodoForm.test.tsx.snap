// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for FilterTodoForm component Should match snapshot 1`] = `
<div>
  <div
    class="inline-block relative w-64"
  >
    <h1>
      Filter
    </h1>
    <select
      class="block appearance-none w-full bg-white border border-gray-400 hover:border-gray-500 px-4 py-2 pr-8 rounded shadow leading-tight focus:outline-none focus:shadow-outline"
    >
      <option
        value="0"
      >
        Marked todos
      </option>
      <option
        value="1"
      >
        Unmarked todos
      </option>
      <option
        value="2"
      >
        All todos
      </option>
    </select>
    <div
      class="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 top-5 text-gray-700"
    >
      <svg
        class="fill-current h-4 w-4"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"
        />
      </svg>
    </div>
  </div>
</div>
`;
